app/app.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     compile(group = "com.google.code.gson", name = "gson", version = "2.8.1")
     fun testFx(name: String, version: String = "4.0.+") =
         create(group = "org.testfx", name = name, version = version)
+    runtime(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "+", classifier = "all")
```

Comment:
this should at least be pinned to v3 so we don't unexpectedly get breaking changes.

I don't understand how the release pipeline versioning affects libraries like ntcore. What is the concern? that we're going to be building with a different version then it was designed for? That's *already* a problem with the current setup. 

Anyway, if it's not currently getting built with the plugins, then we don't need to worry about this until it does, no?